source/2019-12-23-countdown-to-the-new-year-ember-bootstrap-and-ember-paper.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Countdown to The New Year - Ember Bootstrap and Ember Paper
+author: Jared Galanis
+tags: 2019, Addons, DecEmber
+alias: 2019-12-23-countdown-to-the-new-year-ember-bootstrap-and-ember-paper.md
+responsive: true
+---
+
+**This is the 23rd in our DecEmber series–"Countdown to The New Year: 31 Days of Ember Addons". We plan to highlight a new addon each day until the new year, and we hope you'll join us for the fun!**
+
+## Day 23
+
+For the 23rd edition of **Countdown to The New Year** we're going to take a
+look at *two* great design library addons, [`ember-bootstrap`](https://emberobserver.com/addons/ember-bootstrap) and [`ember-paper`](https://emberobserver.com/addons/ember-paper)
+
+READMORE
+
+### Ember Bootstrap - What It Does
+
+`ember-bootstrap` is an addon for using the [Bootstrap](http://getbootstrap.com/) design library in Ember applications. `ember-bootstrap` provides a native Ember component library for some of the more commonly used Bootstrap components.
+
+`ember-bootstrap` can help you can integrate Bootstrap into your application quickly, and in an idiomatic Ember manner.
+
+There are many related Ember addons that integrate with Ember Bootstrap to provide additional features on top of Ember Bootstrap, such as form validation and changesets.
+
+### Ember Bootstrap  - Why I Like It
+
+If you're looking to add Bootstrap to an Ember application, `ember-bootstrap` makes doing so incredibly straight-foward.
+
+`ember-bootstrap`'s clean API's and native Ember components also provide an ergonomic way to use Bootstrap in your Ember application, and all without using Bootstrap's JavaScript. 
+
+Writing forms or modals with these components is a very enjoyable experience and has streamlined the code in several of my projects.
+
+Also, recent versions of `ember-bootstrap` have added additional support for improving accessibility by using ember-focus-trap to implement focus trap for modals and keyboard navigation of dropdowns. `ember-bootstrap` has even added ember a11y tests to the addon's test suite!
+
+`ember-bootstrap` is Octane ready and has refactored it's own components internally to use native classes and angle bracket syntax.
+
+`ember-bootstrap` has excellent documentation, which you can read up on [here](https://www.ember-bootstrap.com/) to learn more about how to use the addon.
+
+### Ember Paper - What It Does
+
+If you're looking for a solution to add Google's [Material](https://www.google.com/design/spec/material-design/introduction.html) design library to an Ember application you can use `ember-paper` to do so! `ember-paper` aims to encapsulate everything possible in Material in Ember components.
+
+`ember-paper` provides an extensive library of components, from buttons and sliders to selects and modal dialogs.
```

Comment:
```suggestion
The `ember-paper` addon provides an extensive library of components, from buttons and sliders to selects and modal dialogs.
```

Btw, doing this because I don't like it when code starts a sentence. It feels incorrect.